index.html

```diff
@@ -225,7 +253,11 @@ <h2>Frameworks</h2>
         <h2>Skills</h2>
         <span class="fas fa-angle-right"></span>
       </div>
+      </div>
     </div>
+
+
+
     <div class="form-section">
```

Comment:
- [ ] In **Desktop version** ,The `Contact me` section should have same design as original website. Kindly fix the `top-left design` and background image of the `contact me` section. **Hint** You can do that by using `border radius` for design.
- Your website 
![gerge](https://user-images.githubusercontent.com/26767383/154677436-44cbd7ee-34ae-4413-85f8-9f5c192846d5.PNG)

- Original website
![gwg](https://user-images.githubusercontent.com/26767383/154677525-d9ef82c9-2331-44b9-92bf-7534c4585407.PNG)